Fix csv examples (#3345)
influxdata/docs-v2: content/influxdb/v2.0/write-data/developer-tools/csv.md

```

{{% note %}}
#### Required annotations and columns

> To write CSV data to InfluxDB with Flux, ALL of the following annotations and columns must be present.

Reviewer: ```suggestion
To write CSV data to InfluxDB with Flux, _all_ of the following annotations and columns must be present.
```

> To write CSV data to InfluxDB with Flux, _all_ of the following annotations and columns must be present.
> Required annotations:

Reviewer 2: ```suggestion
To write CSV data to InfluxDB with Flux, you must include _all_ of the following annotations and columns:
```

- `datatype`
- `group`
- `default`

> See [annotations](/influxdb/cloud/reference/syntax/annotated-csv/#annotations) for more information.

Reviewer: ```suggestion
See [annotations](/influxdb/v2.0/reference/syntax/annotated-csv/#annotations) for more information.
```

> With Flux, there must also be a comma between the annotation name and the annotation values (this differs from the `influx write` command).

Reviewer 2: With Flux, you must also include a comma between the annotation name and the annotation values (this differs from the `influx write` command).

> An example of valid Flux syntax can be found [here](/influxdb/cloud/reference/syntax/annotated-csv/#annotated-csv-in-flux).

Reviewer 2: ```suggestion
See an example of valid syntax for [annotated CSV in Flux](/influxdb/v2.1/reference/syntax/annotated-csv/#annotated-csv-in-flux).
```
